site/docs/pages/token/token-selector.mdx

```diff
@@ -0,0 +1,109 @@
+import { TokenSelector } from '../../../../src/token/components/TokenSelector';
+import App from '../App';
+
+# `<TokenSelector />`
+
+### Warning: component under development, may change in coming weeks
+
+`TokenSelector` component is stylized button component that displays the token info or placeholder text.
```

Comment:
I read this
```
stylized button component that displays the token info or placeholder text.
```
and I am like, "you just described the TokenChip", I understand the selector might have more power, but so far I don't see it. 🤔